internal/workload/manager.go

```diff
@@ -414,6 +420,17 @@ func (w *WorkloadManager) deleteTable() error {
 	return nil
 }
 
+func (w *WorkloadManager) removeServicesFile() error {
+	log.Infof("deleting services file. DeviceID: %s;", w.deviceId)
+	err := w.workloads.RemoveServicesFile()
+	if err != nil {
+		log.Error(err)
```

Comment:
seems there will be duplicated error in the logs if it is being printed here and in the method that may produce the error.